yabt/dot.py

```diff
@@ -59,6 +60,34 @@ def get_not_buildenv_targets(build_context):
     return visited
 
 
+def dot_file_line_proc(nx_g, line):
+    """ Parsing dot file line """
+    l_s = re.split(r'\s*->\s*', line)
```

Comment:
I don't understand what does l_s, d_l, a_l mean. Can you please give them a better name?